packages/xsnap/src/xsnap.c

```diff
@@ -1310,6 +1320,22 @@ static int fxWriteNetString(FILE* outStream, char prefix, char* buf, size_t leng
 	return 0;
 }
 
+static int fxWriteAndResetMeterStats(xsMachine *the, FILE* outStream) {
+#if mxMetering
+  char meterIndexBuf[32];
```

Comment:
again, consistency suggests tabs for indentation

Reply:
I wouldn't say no to a PR that reindented the entire file with spaces...

Reply:
> I wouldn't say no to a PR that reindented the entire file with spaces...

I'd suggest creating a `.clang-format` file and enforcing the formatting of our C code.

But that's for the future.  For now, I'm going for consistency.

Reply:
> I wouldn't say no to a PR that reindented the entire file with spaces...

Using a style that's inconsistent with Moddable's seems like a bad idea. In particular, it suggests that we're going to be writing / maintaining amounts of code comparable to what they produce, which is not a direction I want to head.